coalib/coala_main.py

```diff
@@ -100,34 +100,35 @@ def run_coala(console_printer=None,
     """
     all_actions_possible = provide_all_actions()
     apply_single = None
+
     if getattr(args, 'single_action', None) is not None:
         while True:
             for i, action in enumerate(all_actions_possible, 1):
                 console_printer.print(format_lines('{}'.format(
-                    action), symbol='['))
+                    action), symbol='[', line_nr=' ' + str(i) + '. '))
 
             line = format_lines(STR_ENTER_NUMBER, symbol='[')
-
             choice = input(line)
+            choice = str(choice)
 
-            if choice.isalpha():
+            if choice.isalnum():
                 choice = choice.upper()
                 choice = '(' + choice + ')'
-                if choice == '(N)':
+                if choice == '(N)' or choice == '1':
                     apply_single = 'Do (N)othing'
                     break
                 for i, action in enumerate(all_actions_possible, 1):
-                    if choice in action:
+                    index = all_actions_possible.index(action)
```

Comment:
isn't this equal to `i` now?

Reply:
Yeah sorry! I completly forgot about the enumerate functionality. I'll change that.